packages/components/src/flex/README.md

```diff
@@ -1,98 +1,65 @@
 # Flex
 
-Flex is a layout-based component, which provides a convenient way to align user-interfaces.
+`Flex` is a primitive layout component that adaptively aligns child content horizontally or vertically. `Flex` powers components like `HStack` and `VStack`.
 
 ## Usage
 
+`Flex` is used with any of it's two sub-components, `FlexItem` and `FlexBlock`.
+
 ```jsx
-import { Flex, FlexItem, FlexBlock } from '@wordpress/components';
-import { Icon, wordpress } from '@wordpress/icons';
+import { Flex, FlexItem, FlexBlock, Text } from '@wordpress/components/ui';
 
-const Example = () => {
+function Example() {
 	return (
 		<Flex>
 			<FlexItem>
-				<Icon icon={ wordpress } />
+				<Text>Code</Text>
 			</FlexItem>
 			<FlexBlock>
-				<h2>WordPress</h2>
+				<Text>Poetry</Text>
 			</FlexBlock>
 		</Flex>
 	);
-};
+}
 ```
 
-The most common use-case for `<Flex />` would be to automatically align two (or more) child items of varying sizes. By default, `<Flex />` would vertically center align them, and evenly spread the inner content horizontally, starting at the edges.
-
-Child items can be added directly. However, it is recommended that they are wrapped with `<FlexItem />` (which helps normalize layout styles).
-
-![Flex aligning and distributing 2 items of varying heights](https://make.wordpress.org/design/files/2020/06/flex-2-items.png)
-
-Adding more than 2 child items will spread them across with even gaps in between each item.
-
-![Flex aligning 3 items](https://make.wordpress.org/design/files/2020/06/flex-3-items.png)
-
-### Items and Blocks
-
-`<Flex />` provides 2 sub-components:
-
--   `<FlexItem />`
--   `<FlexBlock />`
-
-These components can be used in combination to achieve flexible adaptive layouts.
-
-`<FlexBlock />` components will grow their widths to take up the remaining space within the `<Flex />` wrapper.
-
-![Flex item with block](https://make.wordpress.org/design/files/2020/06/flex-item-block.png)
-
-`<Flex />` automatically provides "gap" spacing in-between all child items. This value can be adjusted using the `gap` component prop.
-
-`<FlexItem />` and `<FlexBlock />` can be used in combination.
-
-![Multiple Flex items with Flex block](https://make.wordpress.org/design/files/2020/06/flex-item-block-item.png)
+## Props
 
-## Sub-Components
```

Comment:
Do we get sub-components undocumented with the changes proposed?

In practice, when every component exposed in the public API is in its own folder, we can create a README.md file that gets automatically exposed to the developer portal:

https://developer.wordpress.org/block-editor/reference-guides/components/

Every README file gets a link in the sidebar menu.

<img width="1182" alt="Screen Shot 2021-04-29 at 08 23 20" src="https://user-images.githubusercontent.com/699132/116509786-2dc6ed00-a8c4-11eb-9fa0-05c04f2762d2.png">

Reply:
Sure! I can move them into their own folders 🙂